<?xml version="1.0"?>
<launch>
	<arg name="name" default=""/>
	<arg name="tf_prefix" value="$(eval arg('name') + '/' if arg('name') else '')"/>
	<param name="tf_prefix" value="$(arg tf_prefix)"/>
	<node ns="$(eval arg('name') or '/')" pkg="rviz" type="rviz" name="rviz"
		args="-d $(find thymioid)/configuration/thymioid-markers.rviz">
	</node>
</launch>
